resources/prosody-plugins/util.lib.lua

```diff
@@ -239,6 +239,18 @@ function is_feature_allowed(session, feature)
     end
 end
 
+-- Utility function to check whether the user should be hidden.
+function is_user_hidden(session)
+    local context_user = session.jitsi_meet_context_user
+    if (context_user ~= nil
+        and (context_user['moderator'] == "true" or context_user['moderator'] == true)
```

Comment:
What is this context moderator? We do not have it anywhere in the code or documented?

Reply:
This should be new. The idea is for this feature to only be available from the JWT since it's very sensitive.

Reply:
`moderator` is a flag from JWT under user object. We only want to have the participant hidden if they are moderator.

Reply:
I don't like that we introduce new field in the jwt that is not mentioned/documented or used anywhere in this repo.
Maybe we need to add a module which respects that and makes a participant a moderator. As the similar module from jitsi-contrib.

Reply:
hmm I would not call `moderator` a new jwt field, I think it is used for quite a while now. Probably until now it was only used client-side?

Reply:
Maybe, I'm missing it, I do not see it anywhere in this repo? Or in the doc about the tokens: https://github.com/jitsi/lib-jitsi-meet/blob/master/doc/tokens.md

Reply:
removed the moderator check